packages/contracts-bedrock/src/L1/SuperchainConfig.sol

```diff
@@ -86,26 +87,27 @@ contract SuperchainConfig is Initializable, ISemver {
 
     /// @notice Initializer.
     /// @param _guardian             Address of the guardian, can pause the OptimismPortal.
-    /// @param _dependencyManager    Address of the dependencyManager, can add a chain to the dependency set.
     /// @param _paused               Initial paused status.
-    function initialize(address _guardian, address _dependencyManager, bool _paused) external initializer {
+    function initialize(address _guardian, bool _paused) external initializer {
         _setGuardian(_guardian);
-        _setDependencyManager(_dependencyManager);
         if (_paused) {
             _pause("Initializer paused");
         }
     }
 
+    // TODO: check if this function makes sense
+    /// @notice Initializes the first portal of the cluster.
+    /// @param _initialPortal The address of the initial portal.
+    function initializePortal(address _initialPortal) external {
```

Comment:
We agreed that for the first version, we would make the auth be "withdrawal OR multisig" and add a new role to superchain config that represents the multisig. This solves the bootstrapping problem and lets us handle the first upgrade to interop in a simple way. In the future, we can integrate more tightly into OPCM where when deploying a cluster, the first portal is auth'd by op-deployer/opcm. cc @maurelian for visibility